common/FileUtils.java

```diff
@@ -472,5 +478,65 @@ public Message readPrototxtFromZipUnchecked(
   public File getFile(String path) {
     return new File(expandHomeDirectory(path));
   }
+
+  /**
+   * Gets file names in path by pattern. The pattern can containing the * wildcard: this wildcard
```

Comment:
If you're using the standard glob format, you can also link to:
https://docs.oracle.com/javase/8/docs/api/java/nio/file/FileSystem.html#getPathMatcher-java.lang.String-